### PR TITLE
Update tabs and create tabs tasks

### PR DESCRIPTION
**Pull request by NikitaBondarevv**

### CODE REVIEW

File: src/taskList/index.js [hunk 1]
```diff
@@ -1,93 +1,119 @@
...
   }
 
   render() {
-    const { todos, inputValue } = this.state
+    const { todos, value, hidden } = this.state
 
     return (
-      <>
-        <input value={inputValue} onChange={this.taskSearch} className={styles.search} placeholder="SEARCH" />
+      <div className={styles.content}>
```
Comment (iliatsymbal) [inline src/taskList/index.js:83]: where is `.content` class in styles?

**Review by iliatsymbal — CHANGES_REQUESTED** [review]

File: src/editableText/index.js [hunk 2]
```diff
@@ -35,14 +35,14 @@ export class EditableText extends Component {
     const { value, hidden } = this.state;
 
     return (
-      <>
+      <div>
```
[the remark this answers is not in the document]
Comment (NikitaBondarevv) [reply src/editableText/index.js:38]: Not needed, empty tags can be used

[the remark this answers is not in the document]
Comment (iliatsymbal) [reply src/editableText/index.js:38]: and why do you need an empty tag?

File: src/editableText/index.js [hunk 3]
```diff
@@ -35,14 +35,14 @@ export class EditableText extends Component {
     const { value, hidden } = this.state;
 
     return (
-      <div>
+      <>
```
Comment (iliatsymbal) [inline src/editableText/index.js:38]: why do you need react fragment here?

File: src/editableText/index.js [hunk 4]
```diff
@@ -35,14 +35,10 @@ export class EditableText extends Component {
...
     return (
-      <>
-        {
-          hidden ? <span onClick={this.showInput}>{value ? value : this.props.text}</span> :
-          <form onSubmit={this.handleBlur}>
-            <input name="text" value={value} onChange={this.setValue} onBlur={this.handleBlur} autoFocus />
-          </form>
-        }
-      </>
+      hidden ? <span onClick={this.showInput}>{value ? value : this.props.text}</span> :
```
Comment (iliatsymbal) [inline src/editableText/index.js:38]: `value ? value : this.props.text` use more appropriate logical operator

**Pull request merged**